scripts/main.sh

```diff
@@ -48,27 +47,29 @@ main() {
   items="$(op::get_all_items)"
   spinner::stop
 
-  selected_item="$(echo "$items" | awk -F ',' '{ print $1 }' | fzf "${fzf_opts[@]}")"
+  readarray -t selection <<< "$(echo "$items" | awk -F ',' '{ print $1 }' | fzf "${fzf_opts[@]}")"
```

Comment:
Seems like `readarray` only exists in bash >5 (I have 3.2 on my 2025 MBP 🤦 ). Can we try to modify it to better fit older versions (like I have) so it won't break?

[@tomoakley comment](https://github.com/yardnsm/tmux-1password/pull/36/files#r1976011362) looks like a good solution.